docs/docs/formatting.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "Formatting"
+route: "/formatting"
+menu: "References"
+---
+
+# File Formatting
+
+Tanka supports formatting for all `jsonnet` and `libsonnet` files using the `tk fmt` command.
+
+By default, the command excludes all `vendor` directories and any files in the immediate directory.
+
+```bash
+# Run for current and child directories. Run this in the root of the project to format all your files.
+$ tk fmt .
```

Comment:
I'd love to see some more examples here:

1. Formatting single files (`tk fmt myFile.jsonnet`)
2. Testing instead if formatted (`tk fmt -t myFile.jsonnet`)
3. Verbose mode (`tk fmt -v`)